### PR TITLE
mayhem integration

### PR DESCRIPTION
**Pull request by evanrichter**

### CODE REVIEW

File: src/lib/gml-parser/fuzz/fuzz_targets/gml_parse.rs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#![no_main]
```
Comment (xansec) [inline src/lib/gml-parser/fuzz/fuzz_targets/gml_parse.rs:1]: This doesn't look to be a part of shadow.

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: src/lib/gml-parser/fuzz/fuzz_targets/gml_parse.rs (hunk 1 above)
Comment (evanrichter) [reply src/lib/gml-parser/fuzz/fuzz_targets/gml_parse.rs:1]: it's included in the main crate [here](https://github.com/shadow/shadow/blob/98f0217a13e643a4f7ca56fb66203be6d3c45031/src/main/Cargo.toml#L22) and used [here](https://github.com/shadow/shadow/blob/98f0217a13e643a4f7ca56fb66203be6d3c45031/src/main/network/network_graph.rs#L31) and elsewhere in that file

**Review by xansec — APPROVED** [review]
Ok, as long as it's part of the main crate. LGTM!

**Pull request merged**